COMETwebapp/SessionManagement/ISessionAnchor.cs

```diff
@@ -122,11 +122,43 @@ public interface ISessionAnchor
         /// <param name="thingsToCreate">List of Things to update in the session</param>
         Task UpdateThings(IEnumerable<Thing> thingsToUpdate);
 
+        /// <summary>

```

Comment:
i don;t think we need to create all these methods on the the ISessionAnchor for each kind of thing we want to query from the cache that is encapsulated ny the ISession interface. Please just get the data from the ISession provided by the SDK and do that in the ViewModel